tests/e2e/test_client.py

```diff
@@ -176,5 +176,49 @@ def test_get_latest_result_with_query_id(self):
         self.assertGreater(len(results), 0)
 
 
+class TestCRUDOps(unittest.TestCase):
```

Comment:
I thought we couldn't test these? Am I missing something from the PR description?

Reply:
Notice that the DuneClient(version='beta/v1') which is enabled until the end of the month for testing and feedback purposes.